src/main/javascript/modules/admin/commands/player-model.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2014 Virtue Studios
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import { EventType } from 'engine/enums';
+import _events from 'engine/events';
+
+import { sendMessage, sendCommandResponse } from 'shared/chat';
+import { runAnim, addSpotAnim } from 'shared/anim';
+
+_events.bindEventListener(EventType.COMMAND_ADMIN, "anim", (ctx) => {
+	var player = ctx.player;
+	var args = ctx.cmdArgs;
+   // if (args.length < 1 || isNaN(args[0])) {
+	//	sendCommandResponse(player, "Usage: "+ctx.syntax+" [id]", ctx.console);
+	//return;
+	//}
+	var animId = parseInt(args[0]);
+	runAnim(player, animId, function () {
+	sendMessage(player, "Finished animation "+animId);
+	});
+	sendCommandResponse(player, "Running animation "+animId, ctx.console);
+});
+
+_events.bindEventListener(EventType.COMMAND_ADMIN, "gender", (ctx) => {
+	var player = ctx.player;
+	var args = ctx.cmdArgs;
+	//if (args[0] = 0) {
+		//player.getModel().setGender(Gender.MALE);
+		//player.getModel().refresh();
+	//} else if (args[0] = 1) {
+		//player.getModel().setGender(Gender.FEMALE);
+		//player.getModel().refresh();
+	//}
+});
+
+_events.bindEventListener(EventType.COMMAND_ADMIN, [ "spot", "spotanim" ], (ctx) => {
+	var player = ctx.player;
+	var args = ctx.cmdArgs;
+   // if (args.length < 1 || isNaN(args[0])) {
+	//	sendCommandResponse(player, "Usage: "+ctx.syntax+" [id] [type]", ctx.console);
+	//return;
+	//}
+	var spotAnimId = parseInt(args[0]);
+	addSpotAnim(player, spotAnimId);
+});
+
+_events.bindEventListener(EventType.COMMAND_ADMIN, [ "hair", "hairstyle" ], (ctx) => {
+	//ctx.player.getModel().setTempStyle(0, parseInt(ctx.cmdArgs[0]));
```

Comment:
You'll probably have to add `getModel` to the `Player` type in `engine/models` - just copy the definition from one of the other methods `Player` has...